index.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <header>
+        	<img src="logo.png">
+        	<h1>Delicioius</h1>
+        	<h2>THE BEST FOOD BLOG ON THE WEB</h2>
+        	<ul>
+        		<li><img src="facebook.png"></li>
+        		<li><img src="twitter.png"></li>
+        		<li><img src="google.png"></li>
+        		<li><img src="instagram.png"></li>
+        		<li><img src="reddit.png"></li>
+        		<li><img src="pinterest.png"></li>
+        		<li><img src="rss.png"></li>
+        		<li><img src="email.png"></li>
+        	</ul>	
+        </header>
+        <article>
+        	<h1>Chocolate Pizza</h1>
+        	<h2>POSTED ON 15 DEC 2013 / DESERTS</h2>
+        	<h2>PRINT</h2>
+
+        	<img src="chocolatepizza.png">
+
+        	<p> For the fig swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, unitl you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely. </p>
+
+        	<p> Ice cream: In a small pot over medium heat, combine milk, and granulated suger until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+
+        	<p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+
+        	<ul>
+        		<li>11/2 cup milk</li>
+        		<li>1/2 cup mascarpone</li>
+        		<li>1/2 tsp pink salt</li>
+        		<li>1 lb Black Mission Figs</li>
+        		<li>1/2 cup brown sugar</li>
+        		<li>2-4 tbsp water</li>
+        		<li>11/2 cups heavy cream</li>
+        		<li>1/3 granulated sugar</li>
+        		<li>2 egg yolks</li>
+        		<li>1 lemon, juiced</li>
+        		<li>2 tbsp butter</li>
+        		<li>1 cup honey roasted pecans, roughly chopped</li>
+        	</ul>
+        </article>
+
+        <aside>
```

Comment:
I would say that the author info is relevant to the article. I don't think an ```aside``` is appropriate here, you could just make it a ```section``` of the ```article```.